package.json

```diff
@@ -3,24 +3,35 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/jest": "^23.3.9",
+    "@types/node": "^10.12.2",
+    "@types/node-sass": "^3.10.32",
+    "@types/react-dom": "^16.0.9",
+    "@types/react-redux": "^6.0.9",
+    "@types/react-router-dom": "^4.3.1",
+    "@types/redux": "^3.6.0",
+    "@types/redux-promise": "^0.5.28",
     "axios": "^0.18.0",
     "firebase": "^5.5.5",
     "formik": "^1.3.1",
     "json-server": "^0.14.0",
     "lodash": "^4.17.11",
-    "node-sass": "^4.9.4",
+    "node-sass": "^4.10.0",
+    "node-sass-chokidar": "^1.3.4",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
     "react-ratings-declarative": "^3.4.1",
     "react-redux": "^5.0.7",
     "react-router-dom": "^4.2.2",
     "react-scripts": "2.0.5",
+    "react-scripts-ts": "^3.1.0",
```

Comment:
I dont think it's required. With CRA2.0 TS is included via babel, also sass

https://reactjs.org/blog/2018/10/01/create-react-app-v2.html